packages/twenty-server/src/main.ts

```diff
@@ -27,6 +28,11 @@ const bootstrap = async () => {
   // Use our logger
   app.useLogger(logger);
 
+  if (Sentry.isInitialized()) {
+    app.use(Sentry.Handlers.requestHandler());
+    app.use(Sentry.Handlers.tracingHandler());
+  }
```

Comment:
Is it really needed to make it works ?

Reply:
Request Handler is apparently required for request scope isolation but I didn't dig into it too much, it's recommended. TracingHandler I'm guessing would guess starts the transaction, we could do that manually in the yoga plugin if we wanted eventually but the higher we start the better imo - so it feels right.

I've thought of doing something more abstract here where the service declares the providers but again I believe it's overkill and will make the code more complex without ever actually being useful